test/e2e/agent/agent_updates_test.go

```diff
@@ -26,7 +26,7 @@ var _ = Describe("VM Agent behavior during updates", func() {
 	})
 
 	Context("updates", func() {
-		It("should update to the requested image", Label("updates"), func() {
+		It("should update to the requested image", Label("updates", "OCP-75523"), func() {
```

Comment:
I think that we can't share the entire internal test case id, but just the number (75523), since it is a public repository. (At least this was a rule in other public projects)

Reply:
I think we may want to add a prefix to those otherwise it's a bit cryptic if somebody from outside redhat comes and sees "75523", what's that supposed to mean.

May be we should use "rh-qe-75523" or "rh-ocp-75523" ? any preference ?